ci-operator/config/terraform-redhat/terraform-provider-rhcs/terraform-redhat-terraform-provider-rhcs-main.yaml

```diff
@@ -240,6 +284,50 @@ tests:
       WAIT_OPERATORS: "false"
     test:
     - ref: rhcs-e2e-general-tests
+- as: rosa-hcp-advanced-critical-high-presubmit
+  skip_if_only_changed: ^(LICENSE|OWNERS|README\.md|\.gitignore|\.goreleaser\.yaml)$|^docs/|^subsystem/|^examples/
+  steps:
+    cluster_profile: aws-terraform-qe
+    env:
+      CASE_LABEL_FILTER: (Critical,High)&&day1-post&&!NonHCPCluster&&!Exclude
+      CHANNEL_GROUP: candidate
+      CLUSTER_PROFILE: rosa-hcp-ad
+      QE_USAGE: rosa-hcp-advanced-critical-high-presubmit
+      REGION: us-west-2
+      RHCS_ENV: staging
+      WAIT_OPERATORS: "false"
+    test:
+    - ref: rhcs-e2e-tests
+    workflow: rhcs-aws-sts
+- always_run: false
+  as: rosa-hcp-advanced-optional-presubmit
+  steps:
+    cluster_profile: aws-terraform-qe
+    env:
+      CASE_LABEL_FILTER: (day1-post,day2)&&!NonHCPCluster&&!Exclude
+      CHANNEL_GROUP: candidate
+      CLUSTER_PROFILE: rosa-hcp-ad
+      QE_USAGE: rosa-hcp-advanced-optional-presubmit
+      REGION: us-west-2
+      RHCS_ENV: staging
+      WAIT_OPERATORS: "true"
+    test:
+    - ref: rhcs-e2e-tests
+    workflow: rhcs-aws-sts
+- always_run: false
+  as: rosa-hcp-advanced-day1-negative-optional-presubmit
+  steps:
+    cluster_profile: aws-terraform-qe
+    env:
+      CASE_LABEL_FILTER: day1-negative&&!NonHCPCluster&&!Exclude
+      CHANNEL_GROUP: candidate
+      CLUSTER_PROFILE: rosa-hcp-ad
+      QE_USAGE: rosa-hcp-advanced-day1-negative-optional-presubmit
+      REGION: us-west-2
+      RHCS_ENV: staging
+      WAIT_OPERATORS: "false"
+    test:
+    - ref: rhcs-e2e-general-tests
```

Comment:
are there negative cases create some resources? If yes and the cases fail, when/where to release the resources?

Reply:
@yasun1 It should all be handled into the test case code directly.

Reply:
@yasun1 I created https://issues.redhat.com/browse/OCM-6838 to enable day1-negative. For now I am going to remove those jobs from here